scripts/squad-config

```diff
@@ -0,0 +1,502 @@
+            method = requests.post
+        if change_type == "modify":
+            method = requests.put
+
+        for key in config_dict:
+            url = urlunsplit((self.url_scheme, self.base_url, "api/%s" % key, None, None))
+            if not url.endswith("/"):
+                url = url + "/"
+            for item in config_dict[key]:
+                if change_type == "modify":
+                    url = url + str(item['id']) + "/"
+
+                # replace all Null values with empty strings as DRF requires
+                for key, value in item.items():
+                    if value is None:
+                        item[key] = ""
+                    if isinstance(value, dict):
+                        dict_value = item.pop(key)
+                        for k, v in dict_value.items():
+                            new_key = "%s[%s]" % (key, k)
+                            item.update({new_key: v})
+                pprint(item)
+                result = method(url, data=item, headers=self.headers)
+                if result.status_code == 200:
+                    logger.info("Change successfully deployed")
+                    logger.info("%s with ID: %s" % (key, item['id']))
+                else:
+                    logger.warning(result.status_code)
+                    logger.warning(result.text)
+
+    def apply_changes(self,
+                      add_config_items,
+                      modify_config_items):
+        self.__apply_change(add_config_items, "add")
+        self.__apply_change(modify_config_items, "modify")
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-c",
+                        "--config-files",
+                        nargs="+",
+                        default=[],
+                        help="Instance config files",
+                        dest="config_files")
+    parser.add_argument("-d",
+                        "--dru-run",
+                        action="store_true",
+                        default=False,
+                        help="Dry run",
+                        dest="dry_run")
+    parser.add_argument("-v",
+                        "--debug",
+                        action="store_true",
+                        default=False,
+                        help="Enable debug",
+                        dest="debug")
+    parser.add_argument("-u",
+                        "--url",
+                        help="SQUAD instance URL",
+                        dest="url")
+    parser.add_argument("-s",
+                        "--save-to-file",
+                        default=None,
+                        help="Save current config to file",
+                        dest="save_filename")
+    parser.add_argument("-y",
+                        "--assume-yes",
+                        action="store_true",
+                        default=False,
+                        help="Assume 'yes' answer for all questions",
+                        dest="assume_yes")
```

Comment:
this option is not used for anything

Reply:
Arhg, it was used initially. It's required for using the script in non-interactive mode. I'll fix it.